CHANGELOG.md

```diff
@@ -1621,4 +1621,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3778](https://github.com/open-telemetry/opentelemetry-python/pull/3778))
 - Fix license field in pyproject.toml files
   ([#3803](https://github.com/open-telemetry/opentelemetry-python/pull/3803))
-
+- Standardizing timeout calculation in measurement consumer collect to nanoseconds
```

Comment:
@mdcg could you please move the entry to the "Unreleased" section at the beginning of the file?